clustered/integration-test/build.gradle

```diff
@@ -27,7 +27,8 @@ dependencies {
   testCompile project(':dist')
   testCompile project(':clustered:clustered-dist')
   testCompile project(':management')
-  testCompile "org.terracotta.management.dist:management-client:$parent.managementVersion"
+  testCompile "org.terracotta.management.dist:mnm-nms:$parent.managementVersion"
```

Comment:
much clearer to me 👍  we see here that mnm-nms is only for testing in the context of ehcache